### ISSUE TITLE
Why are the Start and End value multipliers removed from CyclicalScheduler derivatives?

### ISSUE THREAD

**Issue by iamkucuk** [pytorch/ignite#458]
I was trying to use the cosine annealing scheduler, however I found out the multipliers for start and end values are removed from cyclical scheduler. The examples from the docs seems like not up to date. Why these multipliers are removed from Ignite?

Edit: Further explainations
`lr_scheduling_handler = CosineAnnealingScheduler(optimizer, 'lr', lr, lr / 100, len(train_loader), start_value_mult=.1, end_value_mult=.1, save_history=True)`

The line of code above gives that error: __init__() got an unexpected keyword argument 'start_value_mult'

This shouldn't be an issue if we look at the corresponding docs page: https://pytorch.org/ignite/contrib/handlers.html#ignite.contrib.handlers.CosineAnnealingScheduler

If we look into source codes of the CyclicalScheduler class, it can be easily seen that start_value_mult and end_value_mult are not there anymore.

**Comment by vfdev-5** [comment]
@iamkucuk could you please provide concrete example of what is missing/removed and what is not working ?

**Comment by iamkucuk** [comment]
> @iamkucuk could you please provide concrete example of what is missing/removed and what is not working ?

Further explainations added into first post.

**Comment by vfdev-5** [comment]
@iamkucuk please add which version of ignite you are using ?

**Comment by iamkucuk** [comment]
> @iamkucuk I think the problem is that you need to install ignite from master:
> 
> ```shell
> pip install git+https://github.com/pytorch/ignite.git
> ```
> 
> Sorry, our docs reflect `master` version and not the stable one. See #308

Yeah I can confirm that. Maybe you can build another pypi package for the current version. It can be confusing for users :)

**Comment by vfdev-5** [comment]
We have also nightly build pip at https://pypi.org/project/pytorch-ignite-nightly/
See #306

**Comment by iamkucuk** [comment]
> We have also nightly build pip at https://pypi.org/project/pytorch-ignite-nightly/
> See #306

Yeah just saw that but it requires pytorch-nightly. It's make it undesirable for me.

**Comment by vfdev-5** [comment]
@iamkucuk in this case you can install as https://github.com/pytorch/ignite/issues/458#issuecomment-474385775

And if you would like to improve our docs by solving #308, this would be highly appreciated by the community :)

I'll close this issue as solved, feel free to reopen if something is incomplete.
